src/collections/careers/digital-marketing-internship/index.mdx

```diff
@@ -51,7 +51,7 @@ import { Col } from "../../../reusecore/Layout";
         </p>
     </div>
     <div className="qualification-section body-sections">
-        <Col xs={12} lg={6} className="qualification-col">
+        <Col $xs={12} $lg={6} className="qualification-col">
```

Comment:
use of `$` ?

Reply:
It indicates passing of the prop to a styled-component as transient props.

Reply:
Thanks for the clarification... :)